### ISSUE TITLE
Zanskar Agriculture Research 

### ISSUE THREAD

**Issue by mikkokotila** [eka-foundation/home#29]
Carving this out from #27. 

As the title indicates, here we are focused on the agriculture aspect. While I appreciate that there might be other aspects that are interesting/valuable as well, given our limited resources, I suggest that in terms of formal research we focus on this topic for now. 

As the first step, we can conduct a literature research on [zanskar agriculture](https://scholar.google.fi/scholar?hl=en&as_sdt=0%2C5&q=zanskar+agriculture&btnG=&oq=zanskar+agricu). 

Following that, we will identify what gaps we have in our information. Thereafter we can decide how to collect such information. Some ideas that had been discussed: conducting workshops, conducting interviews, digital ethnography, and finding a phd/researcher who is willing to spend several months (or years) in Zanskar in an effort to establish a clear view of the agricultural situation.

**Comment by mikkokotila** [comment]
My friend suggested we contact [Juha Helenius](https://www.helsinki.fi/fi/yliopisto/henkilohaku/juha-helenius-9034435) and [Jukka Kivela](https://tuhat.helsinki.fi/portal/fi/persons/jukka-kivela(00ec1244-aef3-4195-a91e-c696500a5550).html) regarding possible phd students. I will do it once we have the association formalized. 

EDIT: one more is J[enny Ohman](https://www.linkedin.com/in/jenny-%C3%B6hman-4b2768b7/) who is working with some kind of co-operation program related with our interest.

**Comment by mikkokotila** [comment]
I might be better for us to find one in India, because they speak Hindi generally and are in the same country, and there is also a much bigger base to source from. What schools do you think would be suitable?

**Comment by mikkokotila** [comment]
Related with this, the sensor project [Farmism](https://github.com/eka-foundation/farmism) is now moved under Eka github.

**Comment by m-anish** [comment]
I'll start looking. Great progress!

**Comment by thepoeticdream** [comment]
[Association Rigzen Zanskar](https://rigzen-zanskar.org/en/homepage/) has projects in Zanskar that falls in EKA focus.  If don't know them should visit on upcoming trip to learn more

**Comment by m-anish** [comment]
There were many NGOs in Zanskar when we went there. Agree, we should find out more and cultivate relationships. Do you happen to know these guys personally?

**Comment by thepoeticdream** [comment]
No. Would guess many of these NGOs are tackling the same problems so should see what solutions they're coming up with in charting EKA efforts

**Comment by m-anish** [comment]
[Related issue on IIAB project](https://github.com/iiab/iiab/issues/1539)

**Comment by thepoeticdream** [comment]
[In the Lap of the Mountains - Irrigation Systems of Ladakh's Farming Communities](https://www.youtube.com/watch?v=Mhmf1FO4bM8)

Very good presentation on traditional irrigation system made with the collective effort of the village and requires collective effort to maintain and repair the system.

**Comment by thepoeticdream** [comment]
[Catalysts of Change - Ladakh video](https://www.youtube.com/watch?v=3xvDIIBi9Es)
[Catalysts of Change pdf report](https://cdkn.org/wp-content/uploads/2014/08/CatalystsofChange-ProjectReport.pdf)

At 6:30 in video can see the [Sutron Automatic Weather Station](https://www.sutron.com/product/portable-automatic-weather-stations-aws/) setup in Serthi (Sakti) village in Leh.  The weather station was part of the climate school setup in the village meant to provide local weather data to farmers.

**Comment by m-anish** [comment]
Nice. We could either do off the shelf, or deign little PCB's that people can solder themselves. :)

The latter will involve them in the process, but higher risk of things not working.

**Comment by thepoeticdream** [comment]
Talking about things working or not working - would be helpful to know the fate and impact of the weather station and climate school in Serthi (done 2014?) and if the farmers were able to utilize the collected weather data set to improve farming yields.  Believe the data from the weather station was supposed to be interpreted by the Indian Meteorological Department Jammu Kashmir and then disseminated back to the village/farmers.  Knowing what happened will be helpful in figuring out the possibilities for EKA weather node.

**Comment by thepoeticdream** [comment]
[Impact of climatic change on agro-ecological zones of the Suru-Zanskar valley, Ladakh (Jammu and Kashmir), India](https://www.researchgate.net/publication/260319959_Impact_of_climatic_change_on_agro-ecological_zones_of_the_Suru-Zanskar_valley_Ladakh_Jammu_and_Kashmir_India)

Good paper on Zanskar agriculture that identifies Zanskar villages by their agricultural productivity.  Some excerpts below

> In the mountainous region like Zanskar, agriculture is the mainstay of the people. 95% of the population are engaged in this activity through sedentary cultivation and pastoral animal rearing.

> The study region has a very limited area (1.5% of the area) under agriculture activity confined in the river basins and along the river terraces whereas the maximum area of the region is either barren (53.08%) or covered under glacier and peri-glacier activity (43.14%).

> The entire valley of Suru-Zanskar being predominantly mountainous in nature, having climatic condition very harsh and the ecology of the region permitted the man to settle down and carry out simple farming in few suitable pockets which had specific features like relatively low altitude, gentle slopes, adequate soil cover and availability of water for irrigation. Therefore, all the agricultural activities are confined to river valleys during the summer season and the growing season extends for about four months only in a year.

> The region supports the cultivation of selected crops like wheat, Grim (barley) Pea, fodder and Vegetable that grow in a short period. The per hectare output of these crops have increased recently due to the development of hybrid variety of seeds that grow in a short period under harsh climatic conditions with assured water supply. The agricultural productivity of a region depends on the fertility and water holding capacity of the soil, intensity of rainfall, depth of water table, and method of cultivation and irrigation facilities. Of all the important factors playing crucial role in productivity, lack of irrigation is the significant factor and among physical factors affecting crop production, amount of rainfall is the most important one.

> (a) High productivity index: The high productivity index has been observed in the villages of Rugrug, Padum Uftipipting, Sumshadi, Sani, Zangla, Tungdikim, Salapia, Karsha, and Testa. The high productivity in the aforementioned villages is mainly due to improving irrigation system, fertile soil, use of chemical fertilizers, and use of modern inputs and availability of low level land.
 
> (b) Moderate productivity index: The moderate productivity index has been found in Remalakiagam, Rarumony, Kargikh, Techakhasar, Hamiling, Lungimireging, Abran, Phe, and Rantaqusha agrarian villages of the study region. The moderate productivity is mainly due to limited irrigation facilities, small size of holdings and unfavourable climate for the cultivation of Wheat, Grim, Peas and vegetables.

> (c) Low productivity index: The low productivity index has been displayed by the villages of the study region namely Ating, Pipcha, Icher, Akshoo, Chah, and Tungithigan. The low productivity is largely due to lack of assured water supply, undulating topography and use of traditional farming techniques.

**Comment by m-anish** [comment]
@thepoeticdream asked whether a greenhouse can be used for planting Seedlings to be ready ahead of cultivation period?
@m-anish says currently it is not used.

**Comment by m-anish** [comment]
Another idea is half-buried-in-the-ground Greenhouses but there is no water for irrigation (or it is hard to get)

**Comment by m-anish** [comment]
More ideas:
* Explore the work by GoGreenGoOrganic - Seabuckthorn and Potato
* Explore Ice Stupas (being tried out for the very first time in Zanskar this year)

**Comment by thepoeticdream** [comment]
The farm season is 4 months so time is critical.  Having seedlings grown in greenhouses during winter and ready to plant once the ground is ready in spring can give a significant time advantage.  Have read that farmers keep soil stored away to throw onto the snow on the farm plot to accelerate snow melting when planting season approaches.

One idea is a village/regional greenhouse that grows seedlings for farmers in the area - a seedling bank?

Water usage in greenhouse can be efficient with drip watering and moisture can be trapped in the greenhouse and not necessarily evaporate away like outdoors outside.  And with the heat inside the greenhouse, during winter can do snow melt to produce water.

Interesting take on the Ice Stupas touching on water rights
[Ice stupas not cutting ice with some farmers in Ladakh](https://indiaclimatedialogue.net/2018/06/27/ice-stupas-not-cutting-ice-with-some-farmers-in-ladakh/)

**Comment by thepoeticdream** [comment]
[Passive Solar Greenhouse in Ladakh](http://www.solaripedia.com/files/1087.pdf)

Great report that touches on many aspects in deploying greenhouses in Ladakh and it's impact and trying to attain sustainability

**Comment by m-anish** [comment]
Great to have david continuing to research on this. If this starts becoming like a reasonable goal for zanskar, then I will bring the zanskar-2019 tag to it. 

As is, bumping milestone.

**Comment by thepoeticdream** [comment]
[Ladakh Ecological Development Group](http://www.ledeg.org) is mentioned in [Passive Solar Greenhouse in Ladakh](http://www.solaripedia.com/files/1087.pdf) in being local partner NGO with [GERES](geres.eu) for projects in Zangskar including greenhouses.  Conversation with them if we haven't already would be good to plug in to what they're doing in Zangskar.

There are many reasonable goals for zanskar-2019 tag for agriculture but I think what would be good is to talk to the farmers and get their thoughts, their story - the substance of the community.  Believe Zanskar agriculture is in an inflection point with generations of tradition under influence from climate change and intrusion of the outside world.  Understanding that landscape is important for EKA to better navigate it.

**Comment by m-anish** [comment]
@thepoeticdream agree! Lets identify things that are tangible goals for zanskar-2019

**Comment by mikkokotila** [comment]
One thing I've learn in the past is that "access" is really important for local farmers. Can we learn more about it Anish? How do families get their vegetables? How do farmers sell their crop? Are there farm advisers in Zanskar? Where do the farmers get equipment, fertilizer, and other resources?

This is very nice thread. Pinned it.

**Comment by mikkokotila** [comment]
> @thepoeticdream agree! Lets identify things that are tangible goals for zanskar-2019

I think it would be nice to be able to at least a) make sure that data collection from sensors is setup in a meaningful way b) introduce one experimental service in the network that relates somehow to the sensors. 

>  tradition under influence from climate change

This is interesting because basically they are really far away from it, but in some way the ones that are most apparently affected by it. Because it has to do with water, it will be nice if the initial r&d somewhat relates with it.

**Comment by thepoeticdream** [comment]
>I think it would be nice to be able to at least a) make sure that data collection from sensors is setup in a meaningful way b) introduce one experimental service in the network that relates somehow to the sensors.

Measuring water is something I think the whole village will pay attention to as it can be existentially relevant.

_People in Shun Shadey Village in Zanskar had completely abandoned their village because the glacier that fed their village has completely melted. Likewise, the residents of Kumic village in Sham were being re-located by the government’s initiative along the banks of Zanskar River due to water shortage problem._ from [In Conversation with Tundup Angmo](http://www.reachladakh.com/in-conversation-with-tundup-angmo/3058.html)

**Comment by thepoeticdream** [comment]
According to [Indian Meteorological Weather Department Automatic Weather Station](http://www.imdpune.gov.in/aws/aws_index.html) there are 127 automatic weather stations deployed 2006-2007.  And additional 550 automatic weather stations are being deployed 2009 onward described in [ESTABLISHING A NETWORK OF 550 AUTOMATIC WEATHER STATIONS AND 1350 AUTOMATIC RAIN GAUGE STATIONS ACROSS INDIA: SCHEME, SCOPE AND STRENGTHS.](https://www.wmo.int/pages/prog/www/IMOP/publications/IOM-104_TECO-2010/P1_36_Ranalkar_India.pdf)  

The automatic weather stations are based on [Sutron](https://www.sutron.com/documents/100-aws-imd-2006.pdf)

Using Sutron sensors (cost notwithstanding) would make sense for a weather dataset that would be "compatible" with the Sutron based AWS network that the Indian Meteorological Department is using.

**Comment by m-anish** [comment]
Related to #25

**Comment by m-anish** [comment]
Document and understand farming practices within greenhouses.

**Comment by m-anish** [comment]
Another idea: Speak with Angdus-ji to understand challenges better.

**Comment by m-anish** [comment]
Related link - https://www.geres.eu/en/

**Comment by thepoeticdream** [comment]
[Solar Greenhouses for the Trans-Himalayas](http://www.solaripedia.com/files/1086.pdf) is companion  to [Passive Solar Greenhouse in Ladakh](http://www.solaripedia.com/files/1086.pdf)

Theory and construction details for passive solar greenhouse.  10 different design variations depending on setting.  The large south facing transparent window side (glass or polyethelyne film or ???) is a design challenge.  Glass = expensive.  Most greenhouses in the region are plastic film which offers limited lifespan and degrade quickly without UV inhibitors.   Polyethelyne film with UV film is expected to last 5 years.  Also the plastic film has little insulating value at cold nights and it is recommended to cover the plastic film with insulating cover to retain the heat inside greenhouse.

**Comment by thepoeticdream** [comment]
[As Himalayan Glaciers Melt, Two Towns Face the Fallout](https://pulitzercenter.org/reporting/himalayan-glaciers-melt-two-towns-face-fallout)

Stongde village in Zanskar hired Nepali stonemasons to make a stone dam which becomes a frozen pond in winter and then supplies melting water to the village.  It seems the Stongde idea came about hearing of the Ice Stupas.

**Comment by thepoeticdream** [comment]
[The Implementation of the Watershed Development Programme in Zangskar, Ladakh: Irrigation Development, Politics and Society.](https://www.soas.ac.uk/water/publications/papers/file38409.pdf)

Good paper giving insights of the nuances of water in Zangskar and of the 2001 government Water Development Programme that had a troubled implementation.

Some takeaways:

Indian census says there are 25 Zangskar villages.  Village may comprise of settlements that can be divided by large distance.  There are about 100 settlements in Zangskar where a settlement may be small cluster of a couple homes.

Ancestral water rights are extremely important in access to village water.  Closed access to water because of ancestral water rights closes village to outsiders/newcomers.

_"If we have enough (irrigation) water then anything is possible"_ from Reru villager

Concrete/cement irrigation channels do not last long term and considered poor investment as they crack and leak from the freeze/thaw of soil.  

No taxes for 40 years as an exempt community

"Give away" subsidies changing traditionally organized village maintenance.

In Ladakh employment with NGO is often perceived as next best thing to salary and status of a Government employee

Government subsidy of food and kerosene has been institutionalized in some Zanskari villages.

SECPAD funded a winter school at Tongde village with teachers posted there.  Shillingskit settlement children of 7 homes were unable to attend because of distance and winter cold.

**Comment by holta** [comment]
Are any https://en.wikipedia.org/wiki/Desert_greening techniques (anti-desertification techniques) applicable to North India, where glaciers/water might be running out?

Or are these permaculture techniques (e.g. promoted by Geoff Lawton on YouTube & in many other places) not relevant here?

**Comment by thepoeticdream** [comment]
Feel there's a recipe for the region to be concocted out of the ingredients of ideas from anti-desertification and permaculture etc but must be the master chef to intuitively pick the right ones to create the appropriate taste for the region.
